drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -850,6 +850,27 @@ void AudioDriverWASAPI::thread_func(void *p_udata) {
 								r = read_sample(ad->audio_input.format_tag, ad->audio_input.bits_per_sample, data, j * 2 + 1);
 							} else if (ad->audio_input.channels == 1) {
 								l = r = read_sample(ad->audio_input.format_tag, ad->audio_input.bits_per_sample, data, j);
+							} else if (ad->audio_input.channels > 2) {
+								l = r = 0;
```

Comment:
Seems like sample can have full 32-bit precision, since sum of multiple samples is used it probably should use a bigger (`int64_t`) temporary variables for `r` and `l` to avoid overflow.

Reply:
Thank you for the suggestion. I have updated the code to use `int64_t` temporary variables (`ltemp` and `rtemp`) to accumulate sample values, which are then cast back to `int32_t` for the final `l` and `r` values.